packages/opentelemetry-plugin-grpc/src/grpc.ts

```diff
@@ -247,29 +237,23 @@ export class GrpcPlugin extends BasePlugin<grpc> {
               const self = this;
 
               const spanName = `grpc.${name.replace('/', '')}`;
+              const parentSpan = plugin._getSpanContext(call.metadata);
```

Comment:
Is this calling a private function on the plugin? If so, why does it need to do that?

Reply:
It only needs it because it needs the binary format parser which is a member of `this._tracer`. I suppose it could be made a helper function similar to your other proposal, but a tracer (or the binary format parser directly) would need to be passed in instead. What do you think?